f/fcs/fn_fcsLocalWarning.sqf

```diff
@@ -0,0 +1,18 @@
+// F3 - FCS/Commander's Override system
+// Credits and documentation: https://github.com/folkarps/F3/wiki
+// Originally from: https://github.com/NikkoJT/njt_fcs (integrated with permission)
+
+// This function shows HUD indicators when the commander's override either lacks a target or is in operation.
+
+params ["_text","_delay"];
+
+if (cameraView == "GUNNER") then {
```

Comment:
This _probably_ should call [disableSerialization](https://community.bistudio.com/wiki/disableSerialization) before calling the UI commands, if I'm reading the wiki right.

Reply:
Normally there's a script error if this is going to be a problem, and this has never caused one. Adding disableSerialization anyway though.